acceptance/framework/flags/flags.go

```diff
@@ -113,6 +116,9 @@ func (t *TestFlags) Validate() error {
 func (t *TestFlags) TestConfigFromFlags() *config.TestConfig {
 	tempDir := t.flagDebugDirectory
 
+	// if the Version is empty consulVersion will be nil
+	consulVersion, _ := version.NewVersion(t.flagConsulVersion)
```

Comment:
I was going to say something about 'nil' but I cannot think of a better value for it. 😄